### ISSUE TITLE
It would be great to allow type fetching

### ISSUE THREAD

**Issue by cleversprocket** [komoot/photon#132]
I only want my users to search for addresses and administrative locations. It would be great if the API supported this by accepting extra parameters.

**Comment by christophlingg** [comment]
what do you exactly mean with 'administrative locations'?

**Comment by cleversprocket** [comment]
Sorry, what I meant to say was Administrative Declared Places: http://wiki.openstreetmap.org/wiki/Map_Features#Administratively_declared_places

**Comment by cleversprocket** [comment]
Whoops, I didn't mean to close it.

**Comment by christophlingg** [comment]
this is on our todo list. we are preparing the 0.2 release at the moment, once it is ready we will have a look at this one.

duplicate of #68
